docs/concepts/Smapi-Command.md

```diff
@@ -101,44 +119,160 @@ The path params
+</dl>
+
+### create-catalog
+
+Creates a new catalog based on information provided in the request.
+
+`create-catalog` command format:
+
+`$ askx smapi create-catalog <--title <title>> <--type <type>> <--usage <usage>> <--vendor-id <vendor-id>> [-p| --profile <profile>] [--debug]`
+
+**Options**
+
+<dl>
+    <dt>--title <title></dt>
+    <dd markdown="span">[REQUIRED] Title of the catalog.</dd>
+    <dt>--type <type></dt>
+    <dd markdown="span">[REQUIRED] Type of catalog. 
+[ENUM]: AMAZON.BroadcastChannel,AMAZON.Genre,AMAZON.MusicAlbum,AMAZON.MusicGroup,AMAZON.MusicPlaylist,AMAZON.MusicRecording,AMAZON.TerrestrialRadioChannel,AMAZON.AudioRecording.</dd>
+    <dt>--usage <usage></dt>
+    <dd markdown="span">[REQUIRED] Usage of the catalog. 
+[ENUM]: AlexaMusic.Catalog.BroadcastChannel,AlexaMusic.Catalog.Genre,AlexaMusic.Catalog.MusicAlbum,AlexaMusic.Catalog.MusicGroup,AlexaMusic.Catalog.MusicPlaylist,AlexaMusic.Catalog.MusicRecording,AlexaMusic.Catalog.TerrestrialRadioChannel,AlexaTest.Catalog.AudioRecording.</dd>
+    <dt>--vendor-id <vendor-id></dt>
+    <dd markdown="span">[REQUIRED] ID of the vendor owning the catalog.</dd>
+    <dt>-p, --profile <profile></dt>
+    <dd markdown="span">The ASK CLI profile to use. When you don't include this option, ASK CLI uses the default profile.</dd>
+    <dt>--debug</dt>
+    <dd markdown="span">When you include this option, ASK CLI shows debug messages in the output of the command.</dd>
+</dl>
+
+### get-catalog
+
+Returns information about a particular catalog.
+
+`get-catalog` command format:
+
+`$ askx smapi get-catalog <-c|--catalog-id <catalog-id>> [-p| --profile <profile>] [--debug]`
 
 **Options**
 
 <dl>
     <dt>-c,--catalog-id <catalog-id></dt>
     <dd markdown="span">[REQUIRED] Unique identifier of the catalog.</dd>
-    <dt>--number-of-upload-parts <number-of-upload-parts></dt>
-    <dd markdown="span">[REQUIRED] The number of parts the file will be split into. An equal number of presigned upload urls will be generated in response to facilitate each part's upload.</dd>
     <dt>-p, --profile <profile></dt>
     <dd markdown="span">The ASK CLI profile to use. When you don't include this option, ASK CLI uses the default profile.</dd>
     <dt>--debug</dt>
     <dd markdown="span">When you include this option, ASK CLI shows debug messages in the output of the command.</dd>
 </dl>
 
-### create-catalog-upload
+### list-catalogs-for-skill
 
-Create a new upload for a catalog and returns location to track the upload process.
+Lists all the catalogs associated with a skill.
 
-`create-catalog-upload` command format:
+`list-catalogs-for-skill` command format:
 
-`$ askx smapi create-catalog-upload <-c|--catalog-id <catalog-id>> <--catalog-upload-request-body <catalog-upload-request-body>> [-p| --profile <profile>] [--debug]`
+`$ askx smapi list-catalogs-for-skill [--next-token <next-token>] [--max-results <max-results>] <-s|--skill-id <skill-id>> [-p| --profile <profile>] [--debug]`
```

Comment:
shouldn't this be just `ask`? change wherever applicable.

Reply:
good call, changed in the new commit